Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentProducer/IncidentProducer.cs

```diff
@@ -235,9 +237,8 @@ private static string GetSecret(string secretName, ILogger log)
             catch  (Exception ex)
             {
                 log.LogInformation("GetSecret ex --> " + ex.Message);
+                return  null;
             }
-            throw new Exception();
```

Comment:
This exception re-throwing will never happen because the function will always exit earlier. You can remove it.